draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2116,7 +2116,281 @@ uCDN-A                         dCDN-B                         dCDN-C
 	<c>RFCthis</c>
 	</texttable>
 	</section>
-	
+
+	<section title="ObjectList" anchor="object-list"><t>
```

Comment:
I'm not sure that I understand why this was moved from the previous section 7.5.2 (maybe becoming a 7.6), to 6.2.x, which which is reserved for top-level objects described in Section 6.1.  (At least this is how it looks without re-rendering the document.)

Reply:
Because of the upcoming 2nd PR that will expand the use of ObjectList beyond trigger spec into the top-level CI/T objects